blackbox/docs/sql/reference/set.txt

```diff
@@ -5,48 +5,61 @@
 SET / RESET
 ===========
 
-Alter and restore global cluster setting values at runtime.
-For a list of the possible settings see
-:ref:`conf-cluster-settings`. Only settings documented with
+Change and restore settings at runtime. To get an overview of available Crate
+settings, see :ref:`configuration`. Only settings documented with
 *Runtime:*  ``yes`` can be changed.
 
-
 Synopsis
 ========
 
 ::
 
-    SET GLOBAL [ PERSISTENT | TRANSIENT ] setting_ident [ = | TO ] value [ , ... ]
+    SET [ SESSION | LOCAL ] setting_ident { = | TO } { value | ident | DEFAULT }
```

Comment:
Maybe we need to distinguish this placeholder value with the one below since for this one we can have 1, 2, 3, ...